README.md

```diff
@@ -112,16 +128,88 @@ Write a conditional statement that...
 8.  Write your javascript solution below
 ```js
 // your answer here
-```
+
+
+var theNumber = 15 ;
+
+if(theNumber % 3 === 0 ){
+  console.log("Fizz");
+}
+else if(theNumber % 5 === 0){
+  console.log("Buzz");
+}
+else if (theNumber % 3 === 0 && theNumber % 5 === 0)
+{
+  console.log("FizzBuzz");
+}
+else {
+  console.log(theNumber)
+}
 
 #### BONUS
 9.  Research a [loop](https://javascript.info/while-for) so that your condition runs on every number from 0 to 100
-```js
-// your answer here
```

Comment:
here is another way you can write this codwhile (a <= 100) {
    if (a%3 == 0 && a%5 == 0 ) {
        console.log("FizzBuzz");   
        a+=1;
    } else if(a%3 == 0) {
        console.log("Fizz");  
        a+=1; 
    } else if (a%5 == 0){
        console.log("Buzz");
        a+=1;
    }
    a+=1;
} e